### ISSUE TITLE
Improve docs landing page

### ISSUE THREAD

**Issue by jpetey75** [lightdash/lightdash#11200]
**Link to existing docs:** https://docs.lightdash.com/

**Changes?**

1. add easy navigation tiles/buttons like this doc https://docs.lightdash.com/guides/cli/intro
2. keep demo video but make it properly embedded, put it smaller and lower on page, and update with new recording that is showing current state of app
3. keep the link to slack community

**Comment by PriPatel** [comment]
Yeah, I think graphics could work. It's more a question about resource tbh! if you could open a ticket outlining how many graphics you need, and what each one needs to convey, I can see how much time it might take.

**Comment by jpetey75** [comment]
The docs have since been rebuilt on a new platform with a redesigned homepage, and the old Docusaurus site this referred to was removed in [#12723](https://github.com/lightdash/lightdash/pull/12723). If you have feedback on the current landing page, let us know and we'll reopen.
